### ISSUE TITLE
Missing extension data

### ISSUE THREAD

**Issue by Jdwalli** [astrowonk/gpxcsv#6]
Recently I exported my GPX data from Apple Health and I noticed that my extensions were not being added to the list. 

```
from gpxcsv import gpxtolist
gpx_list = gpxtolist('route_2021-03-15_1.24am.gpx')
>> [{'lon': -83.239773, 'lat': 42.524461, 'name': 'Route 2021-03-15 1:24am', 'ele': 220.997375, 'time': '2021-03-15T05:10:31Z'}]
```
File Snippet
```
?xml version="1.0" encoding="UTF-8"?>
<gpx version="1.1" creator="Apple Health Export" xmlns="http://www.topografix.com/GPX/1/1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.topografix.com/GPX/1/1 http://www.topografix.com/GPX/1/1/gpx.xsd">
  <metadata>
    <time>2021-11-18T07:33:53Z</time>
  </metadata>
  <trk>
    <name>Route 2021-03-15 1:24am</name>
    <trkseg>
      <trkpt lon="-83.239773" lat="42.524461"><ele>220.997375</ele><time>2021-03-15T05:10:31Z</time><extensions><speed>0.484723</speed><course>269.202515</course><hAcc>1.486173</hAcc><vAcc>1.163164</vAcc></extensions></trkpt>
    </trkseg>
  </trk>
</gpx>
```

**Comment by astrowonk** [comment]
Hey, thanks for posting the issue. What was the original source of the workout? All my exports are from an Apple Watch which put that sort of info under TrackPointExtension, so I hadn't seen this sort of example xml until now. I'll see what I can do with it.

```
<extensions>
					<gpxtpx:TrackPointExtension>
						<gpxtpx:hr>156</gpxtpx:hr>
						<gpxtpx:cad>71</gpxtpx:cad>
						<gpxtpx:speed>0.254569</gpxtpx:speed>
						<gpxtpx:course>289.910889</gpxtpx:course>
						<gpxtpx:hAcc>5.644999</gpxtpx:hAcc>
						<gpxtpx:vAcc>5.373536</gpxtpx:vAcc>
					</gpxtpx:TrackPointExtension>
				</extensions>
```

**Comment by astrowonk** [comment]
Fixed in d28805099ec011454bde8366fdabbb408c4e9a1b

Hopefully I can remember how to upload to pypi...

**Comment by Jdwalli** [comment]
> 

I got this data from the health export. I went to the health app, clicked my profile in the top right, and then exported all my data at the bottom.

**Comment by Jdwalli** [comment]
> Fixed in [d288050](https://github.com/astrowonk/gpxcsv/commit/d28805099ec011454bde8366fdabbb408c4e9a1b)
> 
> Hopefully I can remember how to upload to pypi...

Thank you so much!

**Comment by astrowonk** [comment]
You're welcome! Version 0.2.14 (my version numbers are terrible) is on pypi
/ pip and has the fix.
